### DISCUSSION TITLE
Migrate account1 to account2

### DISCUSSION THREAD

**Discussion by Issam-dz1** [belowaverage-org/SuperGrate#97] [Q&A] [answered]
I have a pc with hostname PC and two accounts account1 and account2 , how to migrate account1 to account2 ?

**Comment by krisdb2009** [comment, marked as answer]
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/1106a189-6b77-4e69-bd3d-ff96c6cc14cd)
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/5a959217-4809-4f1a-8ec9-930142ddf95b)
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/533a2edf-2511-4df6-8cf2-8a2551e5f797)
![image](https://github.com/belowaverage-org/SuperGrate/assets/8226974/611972f9-d8fa-44a7-a2f7-6a225716893e)
